resource/abstractions/individual_access.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Individual Access
+
+General Resource Profile Administration functions for checking an individual user's access to a General Resource Profile. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_my_access()`
+
+```python
+def get_my_access(self, resource: str, class_name: str) -> Union[str, bytes, None]:
+```
+
+#### 📄 Description
+
+Check **Your Access** for a general resource profile.
+
+#### 📥 Parameters
+* `resource`<br>
+  The **general resource profile** for which RACF should check the current user's access level.
+
+* `class_name`<br>
+  The name of the **class** the general resource profile being checked belongs to.
+
+#### 📤 Returns
+* `Union[str, bytes, None]`<br>
+  Returns `None` when the general resource profile has no **Your Access** defined, otherwise returns the access level as a string. If the `ResourceAdmin.generate_requests_only` class attribute is set to `True`, **concatenated Security Request XML bytes** will be returned.
```

Comment:
The wording here is a little strange. Does the following work?

```markdown
Returns `None` when the general resource profile has no **Your Access** section defined,
```

Reply:
Make similar updates to `get_user_access()`

Reply:
Good call, done, and added `for the specified userid` to get_user_access as this section will differ from user to user.

Reply:
also decided to add `for the running userid` to get_my_access.